src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -2099,22 +2099,24 @@ private List<ProjectRootElement> ExpandAndLoadImports(string directoryOfImportin
             // 2. 1 or more project files *found* but ignored (like circular, self imports)
             foreach (var extensionPath in fallbackExtensionPaths)
             {
-                if (!Directory.Exists(extensionPath))
+                string extensionPathExpanded = _data.ExpandString(extensionPath);
```

Comment:
The error message emitted when no project file is found (`ThrowForImportedProjectFromExtensionsPathNotFound`) will not show the fully expanded search path, IIUC. `ThrowForImportedProjectFromExtensionsPathNotFound` needs to be updated, if that is desired.